flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/TwoPhaseCommitSinkFunction.java

```diff
@@ -320,9 +320,9 @@ public void initializeState(FunctionInitializationContext context) throws Except
 
 		state = context.getOperatorStateStore().getListState(stateDescriptor);
 
+		boolean recoveredUserContext = false;
```

Comment:
is this relevant to the bug fix or is this a clean up refactor?

Reply:
This is relevant, there is a semantical difference. Small, but important.